pyogrio/raw.py

```diff
@@ -429,6 +435,7 @@ def open_arrow(
         Meta is: {
             "crs": "<crs>",
             "fields": <ndarray of field names>,
+            "dtypes": <ndarray of numpy dtypes corresponding to fields>
```

Comment:
(definitely not for this PR ;) but was just thinking that we should maybe also return here some (string) repr of the actual GDAL types, instead of this lossy conversion to numpy dtypes)

Reply:
This information was already added to meta in #556, but I forgot to update this docstring apparently.

As it is just adding a few lines of docstrings I included it in this PR.